### DISCUSSION TITLE
Objsearch V0.26.1 crashes Open CPN when upgading, cannot use plugin tab after crash

### DISCUSSION THREAD

**Discussion by WRDRJ** [bareboat-necessities/lysmarine_gen#423] [Q&A]
Objsearch V0.26.1 crashes Open CPN when upgrading, cannot use plugin tab after crash
[https://github.com/nohal/objsearch_pi/issues/28]

**Comment by mgrouch** [comment]
Objsearch included into BBN OS doesn’t crash?
Only the one after plugin upgrade from OpenCPN plugin repository causes crashes?

**Comment by WRDRJ** [comment]
On 2024-06-21 14:14, Mikhail Grushinskiy wrote:
> Objsearch included into BBN OS doesn’t crash?
> Only the one after plugin upgrade from OpenCPN plugin repository
> causes crashes?
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe
> [2].
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
> 
> Links:
> ------
> [1] 
> https://github.com/bareboat-necessities/lysmarine_gen/discussions/423#discussioncomment-9842713
> [2] 
> https://github.com/notifications/unsubscribe-auth/AF5XGVZSMTAARADZECGXP6TZIRURVAVCNFSM6AAAAABJWQHURWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TQNBSG4YTG

Don't know if the one included with Bareboats V -1.0 crashes or not. I 
was just updating all of them. As soon as I upgrade via the plugin tool 
it crashes. Then O runs, but if you go to the plugin tool again, it 
crashes.

Just installed a new downloaded image and am trying it on a different Pi 
with no extra HW
answered the 2 Open CPN questions
plugins
update catalog
update Objsearch, it downloads and installs, crashes Open CPN

do you know where the log file might be.

thanks,

**Comment by mgrouch** [comment]
There numerous plugins that are broken in official OpenCPN repository for pi bookworm. 
I’d recommend instead of blindly upgrading to try using the version preinstalled with BBN OS images. 

Apart from objsearch I’ve heart o-chart from update catalog was broken too.

**Comment by mgrouch** [comment]
To produce log you would need to run OpenCPN with log debug level option on command line
